src/platforms/common/server/cpu_copy_output_surface.cpp

```diff
@@ -67,9 +71,24 @@ class GLHandle
 using RenderbufferHandle = GLHandle<&glGenRenderbuffers, &glDeleteRenderbuffers>;
 using FramebufferHandle = GLHandle<&glGenFramebuffers, &glDeleteFramebuffers>;
 
-auto ensure_context_current(EGLDisplay dpy, EGLContext ctx)
+auto create_current_context(EGLDisplay dpy, EGLContext share_ctx)
     -> EGLContext
 {
+    static const EGLint context_attr[] = {
+        EGL_CONTEXT_CLIENT_VERSION, 2,
+        EGL_NONE
+    };
+
+    auto egl_extensions = eglQueryString(dpy, EGL_EXTENSIONS);
+    if (strstr(egl_extensions, "EGL_KHR_no_config_context"))
+    {
+        // We do not *strictly* need this, but it means I don't need to thread a GLConfig all the way through to here.
+        BOOST_THROW_EXCEPTION((std::runtime_error{"EGL implementation missing necessary EGL_KHR_no_config_context extension"}));
+    }
```

Comment:
Am I reading this wrong, or are we throwing when "EGL_KHR_no_config_context" **is** found and report that it is missing?

Reply:
Empirically, I read this right: fixed